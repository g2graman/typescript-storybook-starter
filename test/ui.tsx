import * as React from "react";
// import { Provider } from "react-redux";
import { create } from "react-test-renderer";

import { genDateValue } from "@vx/mock-data";

// import { StateReducer } from "../src/components/widget/state/reducers";
// import configureStore from "../src/components/widget/state/store";
import Graph, { IPoint } from "../src/components/stateless-graph/stateless-graph";

/*const store = configureStore({
        state: null,
    },
    StateReducer,
);*/

test("Say my name, say my name...", () => {
    const tree1 = create(
        <Graph graphData={genDateValue(20) as IPoint[]} />,
    ).toJSON();
    expect(tree1).toMatchSnapshot();
});
