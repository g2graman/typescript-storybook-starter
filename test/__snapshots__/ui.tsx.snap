// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Say my name, say my name... 1`] = `
<div
  className={undefined}
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <div
    onClick={[Function]}
    style={
      Object {
        "cursor": "pointer",
        "height": "100vh",
        "width": "100%",
      }
    }
  >
    <svg
      height={0}
      width={0}
    >
      <defs>
        <linearGradient
          gradientTransform={undefined}
          id="PinkRed"
          x1="0"
          x2="0"
          y1="0"
          y2="1"
        >
          <stop
            offset="0%"
            stopColor="#F54EA2"
            stopOpacity={1}
          />
          <stop
            offset="100%"
            stopColor="#FF7676"
            stopOpacity={1}
          />
        </linearGradient>
      </defs>
      <g>
        <g>
          <path
            className="vx-area-closed"
            d="M0,0L0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0L0,0L0,0L0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0C0,0,0,0,0,0L0,0Z"
            fill="url(#PinkRed)"
            stroke="url(#PinkRed)"
            strokeDasharray={undefined}
            strokeWidth={2}
          />
        </g>
      </g>
    </svg>
  </div>
</div>
`;
